include/lsst/afw/geom/SpanSet.h

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Im not sure which one yet, but one editor on one of my computers seems to be setup to insert a blank line at the beginning of c(++) files. Fixed